src/main/java/bookface/logic/parser/delete/DeleteBookArgumentsParser.java

```diff
@@ -0,0 +1,43 @@
+package bookface.logic.parser.delete;
+
+import static bookface.logic.parser.CliSyntax.PREFIX_AUTHOR;
+import static bookface.logic.parser.CliSyntax.PREFIX_TITLE;
+
+import java.util.stream.Stream;
+
+import bookface.commons.core.Messages;
+import bookface.logic.commands.delete.DeleteBookCommand;
+import bookface.logic.parser.ArgumentMultimap;
+import bookface.logic.parser.ArgumentTokenizer;
+import bookface.logic.parser.ArgumentsParsable;
+import bookface.logic.parser.Prefix;
+import bookface.logic.parser.exceptions.ParseException;
+import bookface.model.book.Author;
+import bookface.model.book.Book;
+import bookface.model.book.Title;
+
+public class DeleteBookArgumentsParser implements ArgumentsParsable<DeleteBookCommand> {
+    @Override
+    public DeleteBookCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TITLE, PREFIX_AUTHOR);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_TITLE, PREFIX_AUTHOR) || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT,
+                DeleteBookCommand.MESSAGE_USAGE));
+        }
+
+        Author author = new Author(argMultimap.getValue(PREFIX_AUTHOR).get());
+        Title title = new Title(argMultimap.getValue(PREFIX_TITLE).get());
```

Comment:
Same as above, there should be a ParseUtil.parseTitle here as well?

Reply:
added as an issue #112